joomla/joomla-cms
# [com_finder] maps view: searchtools ordering, bug corrections and some improvements

Pull Request for New Issues/Improvements.
#### Summary of Changes

This PR normalizes this view by adding the ordering to the searchtools in com_finder maps view. 

Also:
- Corrects some bugs in the searchtools (for instance, https://github.com/joomla/joomla-cms/issues/10061).
- Some code style improvements.
- Some code improvements.
- Normalizes layout with other views.
- Added max level filter.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14763292/4ca4600c-0989-11e6-9389-63fd80038314.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14763293/52f8c56a-0989-11e6-8ac4-11d79d3754ef.png)
#### Testing Instructions
1. Use latest staging and apply patch
2. Go to Components -> Smart Search -> Filters (Create some index)
3. Go to Components -> Smart Search -> Content Maps
4. Test if search, filters, ordering and page limit (aka searchtools) are working properly.
5. Test also if the toolsbar buttons work properly.

@chrisdavenport to normalize this view with other views, i created two levels of content (branches/childs). Can you confirm there can exist more levels in the finder content maps? thanks

The levels are obtined from [this query](https://github.com/andrepereiradasilva/joomla-cms/blob/com_finder-maps-searchtools/administrator/components/com_finder/models/maps.php#L179-L195).
